### ISSUE TITLE
Missing template to render

### ISSUE THREAD

**Issue by gymgnopf** [cvergne/vscode-php-getters-setters#14]
Hey thank you for your awesome work. 

After updating to version 1.4, I get the error 'Missing template to render', when I click on 'Insert all PHP Getter & Setter'.
I dont use custom templates, but I checked the folder '%APPDATA%\Code\User\phpGettersSetters' and didn't see any template files. 

'Insert PHP Getter & Setter' works just fine. 

Thank you for looking at it.

**Comment by cvergne** [comment]
Hi,

That is strange because all commands are using the same render method.
I neither have any template file and it's working fine.

Did you try to restart VSCode ?
The bug is on any file ? Could you give an example ?

**Comment by cvergne** [comment]
Also could you send a screenshot of your extension settings ? (you can right click on the extension in the sidebar then "Extension Settings".

**Comment by gymgnopf** [comment]
Hi, thank you for your answer.
I did restart VSCode but the problem is still there.

![error-message](https://user-images.githubusercontent.com/62284256/142836866-a7b4dd1c-c0a0-4c9b-b45c-2ef70955c328.png)
.
![settings](https://user-images.githubusercontent.com/62284256/142837096-5216bd53-9214-4d1b-a68a-877ed95c4eea.png)

I hope this is helpful.

**Comment by cvergne** [comment]
Everything seems normal at this point.
Do you have `phpGettersSetters.templatesDir` in `settings.json`? (if you click on "Edit in settings.json")

**Comment by gymgnopf** [comment]
No, there was no entry.

**Comment by alcalyn** [comment]
Same here, works for "Insert PHP getter", "Insert PHP setter", "Insert PHP getter & setter", **but** does not works for "Insert ALL PHP getter & setter".

I tried to define a `phpGettersSetters.templatesDir` with my custom getter and setter templates, but I get same result.

**Comment by cvergne** [comment]
@alcalyn Do you have an example of PHP code not working on your side please ? :)

**Comment by alcalyn** [comment]
Sure, I can reproduce it with a very simple class:

``` php
<?php

class Hello
{
    private int $number;

    private DateTime $date;
}
```

**Comment by cvergne** [comment]
So, should be fixed on v1.6.0. Thanks :)
